docs/getting-started-guides/vsphere.md

```diff
@@ -5,27 +5,77 @@ assignees:
 
 ---
 
-The example below creates a Kubernetes cluster with 4 worker node Virtual
-Machines and a master Virtual Machine (i.e. 5 VMs in your cluster). This
-cluster is set up and controlled from your workstation (or wherever you find
-convenient).
+This page covers how to get started on deploying kubernetes on vSphere and details for how to configure the vsphere cloud provider.
 
 * TOC
 {:toc}
 
-### Prerequisites
+### Getting started with vSphere
 
-1. You need administrator credentials to an ESXi machine or vCenter instance with write mode api access enabled (not available on the free ESXi license).
-2. You must have Go (see [here](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/development.md#go-versions) for supported versions) installed: [www.golang.org](http://www.golang.org).
-3. You must have your `GOPATH` set up and include `$GOPATH/bin` in your `PATH`.
+Kubernetes supports ability to consume storage managed by vSphere via the vSphere Cloud Provider and provides installation support via [Kubernetes-anywhere](https://github.com/kubernetes/kubernetes-anywhere).
+
+### Kubernetes Anywhere
+
+Please use release [v1.4.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.4.7) with kube-anywhere to get kubernetes running on top of vSphere with the vSphere cloud provider enabled.
+
+Detailed steps can be found at the [getting started with kubernetes-anywhere on vSphere page](https://github.com/kubernetes/kubernetes-anywhere/blob/master/phase1/vsphere/README.md)
+
+### vSphere Cloud Provider
+
+vSphere Cloud Provider allows consumption of vSphere managed storage within kubernetes. It supports:
+
+1. Volumes 
+2. Persistent Volumes
+3. Storage Classes and provisioning of volumes.
```

Comment:
Shouldn't we have dynamic provisioning instead of "Storage Classes and provisioning of volumes" as these concepts will be closer as kubernetes mentioned.